source/docs/enterprise/vcs/bitbucket.html.md

```diff
@@ -13,46 +13,49 @@ Additionally, plans are run when a pull request is created or updated. Terraform
 Enterprise will update the pull request with the result of the Terraform plan
 providing quick feedback on proposed changes.
 
-## Registering an OAuth Application & Client
+## Setup an Organization to use Bitbucket Cloud
 
-### Creating and Updating a Bitbucket Cloud OAuth Application
+### Create a Bitbucket Cloud OAuth Consumer
 
-You will need to register Terraform Enterprise as an OAuth Application within your Bitbucket Cloud account. Proceed to https://bitbucket.org/account/user/your-username/oauth-consumers/new. Fill out the required information and set the Redirect URI to a placeholder (ie: http://example.com), as you will need to register the Bitbucket Client with Terraform Enterprise prior to receiving this value. Check all of the permission fields that apply to you, and click Save
+You need to register Terraform Enterprise as an OAuth consumer within your Bitbucket Cloud account. Proceed to https://bitbucket.org/account/user/your-username/oauth-consumers/new. Fill out the form with the following information.
 
-Upon saving the application, you will be redirected to https://bitbucket.org/account/user/your-username/api. Scroll down to OAuth Consumers and click on the application you just created. Copy the Key and Secret. Leave this tab open in your browser as you will need to return to it in a moment.
+- **Name**: Terraform Enterprise (or whatever you want)
+- **Callback URL**: Skip this one for now. You will need to come back and fill it in later.
 
-### Creating a Terraform Enterprise OAuth Client
+The following **Permissions** are required:
 
-In a new tab, navigate to https://atlas.hashicorp.com/settings and, in the left-side panel, select the organization that you’d like to administer your Bitbucket connection, then click on “configuration” in the left-side panel.
+- **Account**: Email, Read, Write
+- **Repositories**: Read, Write, Admin
+- **Pull requests**: Read, Write
+- **Webhooks**: Read and write
 
-Within the “Add OAuthClient” pane, select Bitbucket Cloud and fill in your application key and secret. In the base url field, enter the root url of your Bitbucket instance (i.e. https://bitbucket.org). In the API url field, enter the base api url (i.e. https://api.bitbucket.org/2.0). Create the OAuth client.
+Upon saving, you will be redirected to https://bitbucket.org/account/user/your-username/api. Scroll down to the **OAuth consumers** section and click on the consumer you just created. Copy the **Key** and **Secret**. You will need to enter these values into Terraform Enterprise. Leave this tab open in your browser as you will need to return to it in a moment.
```

Comment:
any time it is more than 2 steps, i prefer to use a numbered list so folks don't accidentally miss one. maybe formatted like this?

```
Upon saving, you will be redirected to https://bitbucket.org/account/user/your-username/api. 

1. Scroll down to the **OAuth consumers** section and click on the consumer you just created
2. Copy the **Key** and **Secret**
3. Enter these values into Terraform Enterprise <<NAME OF PAGE>> 

Leave this tab open in your browser as you will need to return to it in a moment.
```